File: web/src/main/java/by/sorface/passport/web/PassportApplication.java
package by.sorface.passport.web;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.EnableAspectJAutoProxy;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableAsync
@EnableAspectJAutoProxy
@SpringBootApplication
public class PassportApplication {

    public static void main(String[] args) {
        SpringApplication.run(PassportApplication.class, args);
    }

}

File: web/src/main/kotlin/by/sorface/passport/web/PassportApplication.kt
package by.sorface.passport.web

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.context.properties.EnableConfigurationProperties

@EnableConfigurationProperties
@ConfigurationPropertiesScan
@SpringBootApplication
class PassportApplication

fun main(args: Array<String>) {
    SpringApplication.run(PassportApplication::class.java, *args)
}
